conda_forge_tick/auto_tick.py

```diff
@@ -223,6 +225,69 @@ def _commit_migration(
             raise
 
 
+@dataclass(frozen=True)
+class _RerenderInfo:
+    """
+    Additional information about a rerender operation.
+    """
+
+    nontrivial_changes: bool
+    """
+    True if any files which are not in the following list were changed during the rerender, False otherwise:
+    1. anything in the recipe directory
+    2. the README file
+    """
+    rerender_comment: str | None = None
+    """
+    If requested, a comment to be added to the PR to indicate an issue with the rerender.
+    None if no comment should be added.
+    """
+
+
+def _run_rerender(
+    git_cli: GitCli, context: ClonedFeedstockContext, suppress_errors: bool = False
+) -> _RerenderInfo:
+    logger.info("Rerendering the feedstock")
+
+    try:
+        rerender_msg = rerender_feedstock(str(context.local_clone_dir), timeout=900)
+    except Exception as e:
+        logger.error("RERENDER ERROR", exc_info=e)
+
+        if not suppress_errors:
+            raise
+
+        rerender_comment = textwrap.dedent(
+            """
+            Hi! This feedstock was not able to be rerendered after the version update changes. I
+            have pushed the version update changes anyways and am trying to rerender again with this
+            comment. Hopefully you all can fix this!
+            @conda-forge-admin rerender
+            """
+        )
+
+        return _RerenderInfo(
+            nontrivial_changes=False, rerender_comment=rerender_comment
+        )
+
+    if rerender_msg is None:
+        return _RerenderInfo(nontrivial_changes=False)
+
+    git_cli.commit(context.local_clone_dir, rerender_msg, all_=True, allow_empty=True)
+
+    # HEAD~ is the state before the last commit
+    changed_files = git_cli.diffed_files(context.local_clone_dir, "HEAD~")
+
+    recipe_dir = context.local_clone_dir / "recipe"
+
+    nontrivial_changes = any(
+        not file.is_relative_to(recipe_dir) and not file.name.startswith("README")
+        for file in changed_files
+    )
+
+    return _RerenderInfo(nontrivial_changes)
```

Comment:
You used a keyword above, but not here.

Reply:
fixed in f8e181ee330320d6f3c77ba37506dc341f57b598